tests/configs.py

```diff
@@ -14,16 +14,18 @@ def run_virtual(cmd):
 
 configs = Variations([dict(name="Update Docker images", command=["make", "images"]),
                       dict(name="Create virtual environment",
-                           command=["python3.7", "-m", "venv", env_name, "--system-site-packages"]),
```

Comment:
What is wrong with reusing system packages?

Reply:
system have some pre-installed packages. Universum can have dependencies which aren't defined in setup.py but installed on CI system. In this case we will not see error from CI but it really is.
virtualenv will use 'system' cache to install packages in case cache still actual.